exercises/bank-account/bank_account_test.go

```diff
@@ -296,6 +296,31 @@ func TestConcDeposit(t *testing.T) {
 	}
 }
 
+// The benchmark operations are here to allow you to try different
```

Comment:
maybe change allow to help
allow doesn't feel quite right, this is more assistance than permission (nitpicky :smile: )